dla_cnn/desi/DesiMock.py

```diff
@@ -65,36 +70,32 @@ def read_fits_file(self, spec_path, truth_path, zbest_path, normalize = False):
         ivar_r = spec[9].data.copy()
         ivar_z = spec[14].data.copy()
         error = 1./np.sqrt(np.hstack((ivar_b,ivar_r,ivar_z)))
-        self.split_point_br = flux_b.shape[1]
-        self.split_point_rz = flux_b.shape[1]+flux_r.shape[1]
+        self.split_point_br = len(flux_b)
+        self.split_point_rz = len(flux_b)+len(flux_z)
         z_qso = zbest[1].data['Z'].copy()
         ra = spec[1].data['TARGET_RA'].copy()
         dec = spec[1].data['TARGET_DEC'].copy()
 
         self.data = {spec_id[i]:{'FLUX':flux[i],'ERROR': error[i], 'z_qso':z_qso[i] , 'RA': ra[i], 'DEC':dec[i], 'DLAS':spec_dlas[spec_id[i]]} for i in range(len(spec_id))}
-
-        if normalize:
-            err_spec = ''
-            for spec,data in self.data.items():
-                rest_wavelength = self.wavelength/(data['z_qso']+1)
-                if rest_wavelength[0]<1450:
-                    test = (rest_wavelength>=1400)&(rest_wavelength<=1500)
-                    data['FLUX'] = data['FLUX']/np.abs((np.median(data['FLUX'][test])))
-                else:
-                    err_spec = err_spec+"spectra'id:%i redshift:%f \n"%(spec,data['z_qso'])
-            assert err_spec == '', "Something wrong with spectrum below, please check them: \n"+err_spec
-            
+        
+        ndlas = []
+        for spec,data in self.data.items():
+            if 1216*(1+data['z_qso'])*(99/100)<3800:
+                ndlas.append(spec)
+        for item in ndlas:
+            del self.data[item]
 
 
-    def get_sightline(self, id, camera = 'all', rebin = False):
+    def get_sightline(self, id, camera = 'all', rebin = False, normalize = False):
```

Comment:
The parameter `normalize` is never used in this method.

Looks like a bug as you then use `_normalize` in the internal `def get_data()` method.